.travis.yml

```diff
@@ -9,8 +9,8 @@ env:
 
 rvm:
   - 2.0.0
-  - 2.1.4
-  - 2.2.0
+  - 2.1.7
+  - 2.2.3
```

Comment:
Oh, I didn't know you could do this. If this works, then yeah, I think this makes sense.